docs/src/pages/de/404.astro

```diff
@@ -0,0 +1,9 @@
+---
+import SplashLayout from '../layouts/SplashLayout.astro';
```

Comment:
@borisv this is pointing to the wrong file. I think this should be `../../layouts/SplashLayout.astro`?

Reply:
Damn, you're so right. Fixed